src/sentry/static/sentry/less/palette.less

```diff
@@ -96,3 +96,22 @@
 @30: lighten(@100, 70%);
 @20: lighten(@100, 80%);
 @10: lighten(@100, 90%);
+
+// mixins for similarity
+.similar-scores {
+  &.low {
+    background-color: @orange;
+  }
+  &.low-med {
+    background-color: mix(@orange, @yellow-orange, 50%);
+  }
+  &.med {
+    background-color: @yellow-orange;
+  }
+  &.med-high {
+    background-color: mix(@yellow-orange, @green, 40%);
+  }
+  &.high {
+    background-color: @green;
+  }
```

Comment:
Do we mean to have similarity-keywords inside `palette.less`? I'd expect this file to just be colors/variables, and similarity-specific declarations would be in `similarity.less` or something.

Just some thoughts, doesn't require any changes if you disagree.